en/views/helpers/paginator.rst

```diff
@@ -121,7 +131,6 @@ Accepted keys for ``$options``:
 
 * ``escape`` Whether you want the contents HTML entity encoded, defaults to
   ``true``.
-* ``model`` The model to use, defaults to :php:meth:`PaginatorHelper::defaultModel()`.
```

Comment:
With the model being inferred from the result set, should we also raise errors when the `model` option is provided and let the developer know that they need to change their usage?

Reply:
There are a lot of methods which used this option. Not really keen on littering them all with the check and message.

We can reconsider adding a check later if we get too many reports of people tripping over this change.